### PR TITLE
FLZ:8231: Upload video

### PR DESCRIPTION
**Pull request by adi-heffer-folloze**

### CODE REVIEW

**Comment by TanyaFolloze** [comment]
Task linked: [FLZ-8231 Events - Simulive - Upload video](https://app.clickup.com/t/3749831/FLZ-8231)

File: src/common/helpers/mediaHelpers.ts
Comment (rotemgrim) [inline src/common/helpers/mediaHelpers.ts:1]: split into 3 files (its too large)

**Review by rotemgrim — CHANGES_REQUESTED** [review]

File: src/common/helpers/mediaHelpers.ts [hunk 1]
```diff
@@ -291,6 +293,78 @@ export class CloudinaryHelper {
...
+                        method: "POST",
+                        headers: {
+                            "X-Unique-Upload-Id": uniqueUploadId,
+                            "Content-Range": contentRange,
+                        },
+                        data: formData,
+                        progressCallback: (bytesLoaded: number) => {
+                            if(!onProgress) return;
+                            
+                            uploadBytesTrace[chunkIndex] = bytesLoaded;
```
Comment (rotemgrim) [inline src/common/helpers/mediaHelpers.ts:338]: try making this calculation with kind of throttling (no need to update the browser ui if there is no change in percentage)

Comment (rotemgrim) [reply src/common/helpers/mediaHelpers.ts:338]: make sure its not very complex (if it is just drop it)

**Review by rotemgrim — CHANGES_REQUESTED** [review]

**Review by maor-folloze — APPROVED** [review]

**Review by rotemgrim — APPROVED** [review]

**Pull request merged**